tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -819,6 +814,16 @@ TfLiteStatus MicroAllocator::CommitStaticMemoryPlan(
   const int32_t* offline_planner_offsets = nullptr;
   TF_LITE_ENSURE_STATUS(
       builder.GetOfflinePlannedOffsets(&offline_planner_offsets));
+
+  // Allocate buffers for variable tensors.
```

Comment:
let's add a comment about why the variable allocation needs to be here. (i.e. something about its interaction with the offline planner)